setup.py

```diff
@@ -490,7 +490,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'python-telegram-bot~=13.0',
 ]
 trino = [
-    'trino>=0.301.0',
+    'trino>=0.301.0,<0.307.0',
```

Comment:
The `trino` latest release causes issues and I think it came up because I added `deprecated`

Reply:
Could you provide some more details on this? Does it cause conflicts or something else?

Reply:
Yes. the trino provider tests were failing but I guess if I didn't add `deprecated` lib it won't surface. I will remove it and instead make a separate PR if it still occurs again